skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/aibot/ChatWindow.kt

```diff
@@ -114,6 +133,41 @@ fun ConversationField(modifier: Modifier = Modifier) {
   }
 }
 
+@Composable
+fun ChatBubble(message: Message, modifier: Modifier = Modifier) {
+  Box(
+    Modifier.fillMaxWidth()
+      .padding(8.dp)
+      .shadow(elevation = 0.5.dp, shape = RoundedCornerShape(25.dp), clip = true)
+      .background(
+        color = if (message.isMe) ChatColors.promptBackground else ChatColors.responseBackground
+      )
+      .padding(8.dp)
+  ) {
+    Text(
+      text = message.text,
+      color = if (message.isMe) ChatColors.userTextColor else ChatColors.responseTextColor,
+      modifier = modifier.padding(8.dp),
+      fontFamily = FontFamily.SansSerif,
+    )
+  }
+}
+
 fun Modifier.enabled(enabled: Boolean): Modifier {
   return this.then(if (enabled) Modifier.alpha(1.0f) else Modifier.alpha(0.38f))
 }
+
+fun callApi(message: String): String {
+  // function set up to call the DevXP API in the future.
+  // right now, just sends back the user input message
+  return (message)
+}
+
+object ChatColors {
```

Comment:
nice 👍

Reply:
nit before you merge: Let's pull this to its own file as I suspect you'll keep adding to it. Would be great if we could pull these from the Jewel theme too